src/Copy/BrandCopy.elm

```diff
@@ -176,17 +176,37 @@ brandCopy key =
 
         PrivacySectionFourList ->
             CopyList
-                [ "On our computers"
-                , "On our survey provider Typeform’s servers. For more information see Typeform's privacy policy."
-                , "In Google Analytics. For more information see Google's privacy policy."
-                , "By our partner CAST on their servers if you have explicitly expressed the desire to share your information with them whilst using our site. For more information see CAST's privacy policy."
+                [ CopyText "On our computers"
+                , CopyWithLink
+                    { textBefore = "On our survey provider Typeform’s servers. For more information see"
+                    , linkText = "Typeform's privacy policy"
+                    , destination = "https://www.typeform.com/help/gdpr-rights-for-respondents/"
+                    , textAfter = "."
+                    }
+                , CopyWithLink
+                    { textBefore = "In Google Analytics. For more information see"
+                    , linkText = "https://support.google.com/analytics/answer/6004245"
+                    , destination = "Google's privacy policy"
+                    , textAfter = "."
+                    }
+                , CopyWithLink
+                    { textBefore = "By our partner CAST on their servers if you have explicitly expressed the desire to share your information with them whilst using our site. For more information see"
+                    , linkText = "CAST's privacy policy"
+                    , destination = "https://www.castsoftware.com/privacy"
+                    , textAfter = "."
+                    }
                 ]
 
         PrivacySectionFiveH3 ->
             CopyText "Your rights"
 
         PrivacySectionFiveP ->
-            CopyText "We guarantee to meet your rights in compliance with GDPR. Find out about those rights from the ICO GDPR indvidual rights guide. To exercise any of your rights, please contact us at hello@contemplatingaction.org.uk"
+            CopyWithLink
+                { textBefore = "We guarantee to meet your rights in compliance with GDPR. Find out about those rights from the"
+                , linkText = "ICO GDPR indvidual rights guide"
+                , destination = ""
```

Comment:
https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/individual-rights/